### ISSUE TITLE
Is this going to be updated?

### ISSUE THREAD

**Issue by Aditya94A** [jakiestfu/Snap.js#284]
Can we get an update to the state of the library? It's got a lot of stars but it also hasn't been updated in a while. Does it still work? Perhaps there could be a list of the most common issues in the readme?

Otherwise can someone please recommend an alternative? I've been searching for a responsive  gesture swipe-able side menu for days now.

**Comment by bpongy** [comment]
Same questions.

**Comment by alexiovay** [comment]
For me it's the best solution to create a hamburger menu out there, even if it's outdated. Nowhere else I found this smooth-swiping experience like this. It feels like an native app then. There is only one bug I discovered which affects Android devices, but there is a solution here in the forums.

**Comment by Aditya94A** [comment]
Yeah but still, the author @jakiestfu seems pretty active but for some reason has been ignoring this repo for four years.

At the very least ownership should be transferred to someone else.

**Comment by mttmccb** [comment]
That would covered in #278 @AdityaAnand1, looks like no one has come forward.
